lib/api_browser/Gruntfile.js

```diff
@@ -327,7 +327,7 @@ module.exports = function(grunt) {
   grunt.registerTask('runGenerator', function() {
     var exec = require('child_process').exec;
     var done = this.async();
-    exec('bundle exec rake praxis:docs:generate', {cwd: userDocsPath + '/../'}, done);
+    exec('bundle exec rake praxis:docs:generate_beta', {cwd: userDocsPath + '/../'}, done);
```

Comment:
This PR also changes generate_beta -> generate, so does this change still need to be here?